src/io/flutter/inspector/InspectorService.java

```diff
@@ -60,6 +61,30 @@ public void connectionClosed() {
     vmService.streamListen("Extension", VmServiceConsumers.EMPTY_SUCCESS_CONSUMER);
   }
 
+  public FlutterDebugProcess getDebugProcess() {
+    return debugProcess;
+  }
+
+  public CompletableFuture<XSourcePosition> getPropertyLocation(InstanceRef instanceRef, String name) {
+    return getInstance(instanceRef).thenComposeAsync((Instance instance) -> getPropertyLocationHelper(instance.getClassRef(), name));
+  }
+
+  public CompletableFuture<XSourcePosition> getPropertyLocationHelper(ClassRef classRef, String name) {
+    return inspectorLibrary.getClass(classRef).thenComposeAsync((ClassObj clazz) -> {
+      for (FuncRef f : clazz.getFunctions()) {
+        if (f.getName().equals(name)) {
```

Comment:
Add a TODO to handle checking for private properties that match the name.
Eg. if name == 'textAlignment' 
we should check if the class has a 
_textAlignment property.

Reply:
Good catch!  Done.